config/default/core.entity_form_display.node.hs_basic_page.default.yml

```diff
@@ -145,5 +152,4 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
-hidden:
-  layout_builder__layout: true
+hidden: {  }
```

Comment:
@codechefmarc  Sorry, I didn't highlighted this in the review, but this needs to be reverted too, it's related to the other change you reverted before.

Reply:
Reverted.